recipes/json-schema-validator/all/conanfile.py

```diff
@@ -23,10 +23,12 @@ class JsonSchemaValidatorConan(ConanFile):
     options = {
         "shared": [True, False],
         "fPIC": [True, False],
+        "JSON_DIAGNOSTICS": [True, False],
```

Comment:
I think it's better to use lowercase options like: `json_diagnostics`
```suggestion
        "json_diagnostics": [True, False],
```